resources/js/components/Elements/Slider.vue

```diff
@@ -42,6 +61,19 @@
                 this.showRight = (this.slider.offsetWidth + this.position) < this.slider.scrollWidth - 1
             },
 
+            setHover(value) {
+                this.hover = value
+            },
+            autoScroll() {
+                if (!this.hover) {
+                    let next = this.currentSlide + 1
+                    while (next >= this.slidesTotal) {
```

Comment:
Shouldn't we just set the next navigation to 0 in case we exceed the total slides available?
Saves us some unnecessary looping code